pykoi/rlhf/supervised_finetuning.py

```diff
@@ -208,15 +287,15 @@ def create_datasets(self, tokenizer, args):
         train_dataset = ConstantLengthDataset(
             tokenizer,
             dataset["train"],
-            formatting_func=self.prepare_sample_text,
+            formatting_func=self.prepare_d2l_text,
```

Comment:
qq: same as my comments above, we should make this configurable to maintain the old functionality.

Reply:
done